test/Table_Tests/src/Common_Table_Operations/Main.enso

```diff
@@ -97,7 +98,9 @@ type Test_Selection
          `Duration`/`Period` type.
        - supports_nanoseconds_in_time: Specifies if the backend supports
          nanosecond precision in time values.
-    Config supports_case_sensitive_columns=True order_by=True natural_ordering=False case_insensitive_ordering=True order_by_unicode_normalization_by_default=False case_insensitive_ascii_only=False take_drop=True allows_mixed_type_comparisons=True supports_unicode_normalization=False is_nan_and_nothing_distinct=True distinct_returns_first_row_from_group_if_ordered=True date_time=True fixed_length_text_columns=False supports_decimal_type=False supports_time_duration=False supports_nanoseconds_in_time=False
+       - supports_mixed_columns: Specifies if the backend supports mixed-type
+         columns.
+    Config supports_case_sensitive_columns=True order_by=True natural_ordering=False case_insensitive_ordering=True order_by_unicode_normalization_by_default=False case_insensitive_ascii_only=False take_drop=True allows_mixed_type_comparisons=True supports_unicode_normalization=False is_nan_and_nothing_distinct=True distinct_returns_first_row_from_group_if_ordered=True date_time=True fixed_length_text_columns=False supports_decimal_type=False supports_time_duration=False supports_nanoseconds_in_time=False supports_mixed_columns=False
```

Comment:
this is getting very long...
I suggest we drop `supports_` from names. Not necessarily in this PR!